examples/bri-3/src/bri/zkp/components/merkleTree/merkleTree.ts

```diff
@@ -0,0 +1,74 @@
+import { Tree } from "./tree"
+import { IMerkleTree} from "./merkleTree.interface";
+import { PreciseProofs } from "ew-precise-proofs-js";
+
+export class MerkleTree implements IMerkleTree {
+  tree: Tree;
+
+  constructor(document: any){
+    const merkleTree: Tree = this.createTree(document);
+
+    this.tree.leaves = merkleTree.leaves;
+    this.tree.root = merkleTree.root; 
+  }
+
+  private createTree(document: any): Tree {
+    try {
+      var tree: Tree;
+
+      const leaves: PreciseProofs.Leaf[] = this.createLeaves(document);
+
+      const merkleTree = PreciseProofs.createMerkleTree(
+        leaves.map((leaf: PreciseProofs.Leaf) => leaf.hash)
+      );
+
+      const root = PreciseProofs.getRootHash(merkleTree);
+      
+      tree.leaves = leaves;
+      tree.root = root;
+
+      return tree;
+    } catch (e) {
+      console.log("Tree not created");
```

Comment:
nitpick but lets use one log console.log(`Tree not created: ${e}`)

Reply:
@skosito you are actually bringing up a good point.

Could we please NOT USE console.log but rather a ts logger e.g. https://www.npmjs.com/package/typescript-logger ... it will make troubleshooting so much easier. Let's agree on an approach and method!! cc @ognjenkurtic @Kasshern @Ybittan @Manik-Jain @biscuitdey

Reply:
Agree completely, console.log should be used only for local development and debugging purposes. Let's discuss the proper approach for logging in a separate issue, i ll open one.